lib/plugin.js

```diff
@@ -34,6 +35,13 @@ const axiosExtra = {
   onError(fn) {
     this.onRequestError(fn)
     this.onResponseError(fn)
+  },
+  create(options) {
+    const instance = Axios.create(defu(options, this.defaults))
+    extendAxiosInstance(instance)
```

Comment:
The configuration of the interceptors should be extended too, no?
https://github.com/nuxt-community/axios-module/blob/dev/lib/plugin.js#L193

Reply:
What do you think @pi0 ?

Reply:
True. Agree. Maybe we can refactor L185:L197 to a create function and also don't duplicate logic

Reply:
I agree

Reply:
Sure, I can do it

Reply:
Done, now it looks better